docs/api/tasks-delete-task

```diff
@@ -0,0 +1,47 @@
+layout: page
+---
+
+# delete task
+
+Delete a task from the to-do service. 
+
+## URL
+
+```shell
+
+{DELETE}{server_url}/tasks/
+```
+
+## Request headers
+
```

Comment:
For this operation, the task `id` is a URL parameter.
See `api/users-get-user-by-id.md` for another example of such an operation.